src/actions/CryptoExchangeActions.js

```diff
@@ -148,7 +148,8 @@ export const shiftCryptoCurrency = () => async (dispatch: Dispatch, getState: Ge
         quoteIdUri,
         payinAddress,
         uniqueIdentifier,
-        supportEmail
+        supportEmail,
+        isEstimate
```

Comment:
I don't see where `exchange_notes_metadata_generic2` gets updated to use `%12$s`.

Reply:
FIxed